src/components/Explore/TokenTable/TokenRow.tsx

```diff
@@ -200,14 +200,14 @@ const PriceInfoCell = styled(Cell)`
 const SortArrowCell = styled(Cell)`
   padding-right: 2px;
 `
-const SortingCategory = styled.span`
-  display: flex;
+const HeaderCellWrapper = styled.span<{ onClick?: () => void }>`
   align-items: center;
-  justify-content: center;
-  cursor: pointer;
-`
-const SortOption = styled.span`
-  cursor: pointer;
+  cursor: ${({ onClick }) => (onClick ? 'pointer' : 'unset')};
```

Comment:
```
  ${({ onClick }) => onClick && `
    cursor: pointer;
  `}
```

Reply:
pref the current implementation because it allows predictable property organization. It's not clear where you'd expect to find the snippet you shared if all the other properties are alphabetized.